src/app/components/mnoe-product-markups-list/mnoe-product-markups-list.coffee

```diff
@@ -111,6 +130,15 @@
         )
       )
 
+    vm.showOrgName = (name) ->
```

Comment:
I'd like to separate the translation from the logic but this is a bit tricky as you have either a translation key or the org name.

Maybe this is a bit cleaner?
```coffeescript
return  $translate.instant("mnoe_admin_panel.dashboard.product_markups.add_markup.modal.all_companies") unless name

if vm.readOnlyView
  $translate.instant("mnoe_admin_panel.dashboard.product_markups.add_markup.modal.customer_specific")
else
  name
```